src/main/jhipster/src/main/webapp/app/entities/study/entry-details/import-entry-details.component.ts

```diff
@@ -303,3 +357,33 @@ export class ImportEntryDetailsPopupComponent implements OnInit, OnDestroy {
         });
     }
 }
+
+@Component({
+    selector: 'jhi-modal-no-entry-values',
+    template: `
+		<jhi-modal [title]="title">
+			<div class="modal-body word-wrap" [innerHTML]="sanitizer.bypassSecurityTrustHtml(message)">
+			</div>
+			<div class="modal-footer">
+				<button (click)="confirm()" class="btn btn-primary" data-test="modalConfirmButton">
+					<span>{{confirmLabel}}</span>
+				</button>
+			</div>
+		</jhi-modal>
+    `
+})
+export class ModalNoEntryValuesComponent {
```

Comment:
Could be a reusable, generic, _ModalAlert_ component..

Reply:
Done! I move this logic to a new ModalAlertComponent